lib/actions.js

```diff
@@ -0,0 +1,5 @@
+function fetchPlants() {
+  return fetch(`http://harvesthelper.herokuapp.com/api/v1/plants?api_key=${"api key"}`).then(response => response.json().then(json => {
```

Comment:
Good call good call.  I have actually removed all of that code and am going back to jquery.